cdap-ui/app/features/hydratorplusplus/controllers/create/create-studio-ctrl.js

```diff
@@ -29,9 +29,17 @@ class HydratorPlusPlusStudioCtrl {
       HydratorPlusPlusNodeConfigStore.reset();
       HydratorPlusPlusConsoleActions.resetMessages();
     });
-
+    let getValidArtifact = () => {
+      let isValidArtifact;
+      if (rArtifacts.length) {
+        isValidArtifact = rArtifacts.filter(r => r.name === rSelectedArtifact);
+      }
+      return isValidArtifact.length? isValidArtifact[0]: rArtifacts[0];
```

Comment:
not sure about the javascript coding style here but should there be spaces before the question marks and colons?